ansible/roles/test/files/ptftests/sflow_test.py

```diff
@@ -193,14 +193,14 @@ def analyze_counter_sample(self, data, collector, polling_int, port_sample):
 
     def analyze_flow_sample(self, data, collector): 
         logging.info("packets collected from interfaces ifindex : %s" %data['flow_port_count'])
-        logging.info("Expected number of packets from each port : %s to %s"%(50*0.6,50*1.4))
+        logging.info("Expected number of packets from each port : %s to %s"%(100*0.6,100*1.4))
```

Comment:
Can you add spacing in between for better readability? For example:
`(100*0.6,100*1.4)` -> `(100 * 0.6, 100 * 1.4)`

Reply:
Done.